{"name": "test_unauthorized_access", "status": "passed", "description": "Проверка отказа в доступе при использовании недействительного API ключа.", "steps": [{"name": "Создание клиента с недействительным API ключом", "status": "passed", "start": 1744809344063, "stop": 1744809344064}, {"name": "Попытка получить задачу", "status": "passed", "steps": [{"name": "Получение задачи по ID: '86c3421f7'", "status": "passed", "steps": [{"name": "GET /task/86c3421f7", "status": "passed", "start": 1744809344064, "stop": 1744809344718}], "parameters": [{"name": "task_id", "value": "'86c3421f7'"}], "start": 1744809344064, "stop": 1744809344718}], "start": 1744809344064, "stop": 1744809344718}], "start": 1744809344063, "stop": 1744809344718, "uuid": "46cb3bb1-ef15-4571-bf48-ca6b6df5eec3", "historyId": "9cba1e9a7f27cb6079dcba3f607f479d", "testCaseId": "9cba1e9a7f27cb6079dcba3f607f479d", "fullName": "tests.api.test_tasks#test_unauthorized_access", "labels": [{"name": "feature", "value": "Авторизация"}, {"name": "parentSuite", "value": "tests.api"}, {"name": "suite", "value": "test_tasks"}, {"name": "host", "value": "DESKTOP-G6JEM9V"}, {"name": "thread", "value": "8820-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api.test_tasks"}]}